src/main/java/org/traccar/protocol/ArknavProtocolDecoder.java

```diff
@@ -47,6 +47,8 @@ public ArknavProtocolDecoder(Protocol protocol) {
             .number("(d+.?d*),")                 // hdop
             .number("(dd):(dd):(dd) ")           // time (hh:mm:ss)
             .number("(dd)-(dd)-(dd),")           // date (dd-mm-yy)
+            .expression(".{4},")                 // Unit Version number
+            .number("(dd)?")                     // Battery level
```

Comment:
It seems to conflict with the existing protocol.

Reply:
Which protocol is it in conflict with?
Mapping is according to Arknav protocol documentation

Reply:
I see other unit tests have `9.00,D3` in this spot. Which means this is incorrect pattern.